### DISCUSSION TITLE
New to PACE-HRH? Post here to get in touch!

### DISCUSSION THREAD

**Discussion by BHagedorn-IDM** [InstituteforDiseaseModeling/PACE-HRH#189] [General]
Welcome to PACE-HRH! Please post your name, organization, and a little bit about your use case and what you are looking to use the package for in this discussion channel.

**Comment by maharsantoso** [comment]
Hi Hagedorn,

I'm new in PACE-HRH, i will install it in Rstudio, but unfortunately there is warning: installation of package ‘D:/Mahar/PACE-HRH/PACE-HRH-1.1.0.tar.gz’ **had non-zero exit status**

can you help me? Thank you
